slidey_js.js

```diff
@@ -0,0 +1,36 @@
+$(document).ready(function(){
+
+backgroundImage = $("#background")
+
+images = [
+  "url('http://i.giphy.com/e34HHm5F08FqM.gif')",
+  "url('http://i.giphy.com/JzIyZB0fY4dPO.gif')",
+  "url('http://i.giphy.com/P62H3xNZL24ow.gif')"
+]
+
+imageArray = images[0]
+
+backgroundImage.addEventListener("loadup", backgroundInterval())
```

Comment:
Careful! You are invoking a function here with the `()` when instead, you should reference a function